lib/init/config-initializer.js

```diff
@@ -100,14 +100,21 @@ function getModulesList(config, installESLint) {
             modules[`eslint-plugin-${plugin}`] = "latest";
         }
     }
-    if (config.extends && config.extends.indexOf("eslint:") === -1) {
-        const moduleName = `eslint-config-${config.extends}`;
-
-        modules[moduleName] = "latest";
-        Object.assign(
-            modules,
-            getPeerDependencies(`${moduleName}@latest`)
-        );
+    if (config.extends) {
+        const extendList = Array.isArray(config.extends) ? config.extends : [config.extends];
+
+        for (const extend of extendList) {
+            if (extend.startsWith("eslint:") || extend.startsWith("plugin:")) {
+                continue;
+            }
+            const moduleName = `eslint-config-${extend}`;
```

Comment:
`extends: ["eslint-config-airbnb"]` is valid, but this line will cause a problem with that case. I highly recommend importing our "naming" utility module and using that to ensure proper normalization of config names.